scripts/documentation/generate-executors-data.ts

```diff
@@ -87,7 +87,7 @@ function generateTemplate(
     Options can be configured in \`${filename}\` when defining the executor, or when invoking it.
     ${
       framework != 'angular'
-        ? `Read more about how to use executors and the CLI here: https://nx.dev/${framework}/getting-started/nx-cli#running-tasks.`
+        ? `Read more about how to use executors and the CLI here: https://nx.dev/latest/${framework}/getting-started/nx-cli#common-commands.`
```

Comment:
I would remove the "latest/${framework}" to have a generic url instead https://nx.dev/getting-started/nx-cli#common-commands.

Reply:
FYI, I had to update `internal-link-checker` to allow missing framework in the link.